kotlin/src/main/resources/detekt.yml

```diff
@@ -160,28 +160,22 @@ empty-blocks:
 exceptions:
   active: true
   ExceptionRaisedInUnexpectedLocation:
-    active: false
+    active: true
     methodNames: [toString, hashCode, equals, finalize]
   InstanceOfCheckForException:
     active: false
```

Comment:
je le metterais à truc ca :P

Reply:
En même temps ça gosse de pas pouvoir faire `catch (e A | B | C) {  ... }` en kotlin.

Reply:
ohhh ça comprend ça!? fuck off alors, met pas ça. J'pensais que c'était juste : 

```
catch (e: A | B) {
  if (e instanceof B) {
  }
}
```

Auquel cas fait 2 catch je dirais. Si juste `(e: A | B)` trigger l'erreur, oublie ce que j'ai dit

Reply:
Pas sur qu'on se comprends. En kotlin, on peut pas faire `(e: A | B)`. Donc on fait souvent 
```kotlin
catch (e: ExceptionGénérique) {
  when (e) ...
}
```
C'est le seul moyen de faire le même traitement pour plusieurs exceptions sans copié collé.

Reply:
https://stackoverflow.com/questions/36760489/how-to-catch-many-exceptions-at-the-same-time-in-kotlin

Reply:
ahhhh j'avais mal compris, pis j'pensais que tu voulais dire que tu pouvais faire un `|` asteur :P mais ouin si y'a rien de mieux, c'est good à false hehe